apps/api/.env.example

```
@@ -30,6 +30,14 @@ ENABLE_ARCHIVED_NOTIFICATION_DELETION=false # Toggle to enable/disable deletion
 DELETE_INTERVAL_IN_SECONDS=86400 # Interval (in seconds) for deletion of archived notifications. DEFAULT: 86400 (every 1 day / daily)
 DELETE_ARCHIVED_NOTIFICATIONS_OLDER_THAN=90d # Use formats from https://github.com/vercel/ms. Use positive values. DEFAULT: 90d (older than 90 days). LIMIT: 10y (not older than 10 years). NOTE: Retry notifications are also deleted based on this age
 
+# Webhook configuration
+WEBHOOK_MAX_RETRY_COUNT=5 # Max webhook delivery attempts before marking the delivery permanently FAILED. DEFAULT: 5
+WEBHOOK_RETRY_INTERVAL=30m # Fixed delay between webhook retry attempts, use formats from https://github.com/vercel/ms. DEFAULT: 30m
+WEBHOOK_LOG_RETENTION_DAYS=60 # Days to keep notify_webhook_logs rows before the cleanup endpoint deletes them. Leave empty to disable cleanup (unbounded growth). DEFAULT: 60
+WEBHOOK_LOG_CLEANUP_INTERVAL_IN_SECONDS=86400 # Interval (in seconds) for scheduler.sh to call the webhook log cleanup endpoint. DEFAULT: 86400 (every 1 day / daily)
+WEBHOOK_REQUEST_TIMEOUT_MS=10000 # Max time (in ms) to wait for a webhook partner's response before treating the attempt as failed. DEFAULT: 10000 (10s)
+SCHEDULER_INTERNAL_KEY=your-scheduler-secret # Shared secret scheduler.sh sends via x-scheduler-key to call internal endpoints (e.g. webhook log cleanup). WARNING: Required in production, min 32 characters recommended
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SCHEDULER_INTERNAL_KEY key should go before the SERVER_PORT key

(UnorderedKey)